src/bootstrap/config.rs

```diff
@@ -636,6 +636,12 @@ impl Config {
                     let target = self.target_config.entry(target).or_insert(Target::default());
                     target.qemu_rootfs = Some(parse_configure_path(value));
                 }
+                "CFG_QEMU_AARCH64_ROOTFS" if value.len() > 0 => {
+                    let target = "aarch64-unknown-linux-gnu".to_string();
```

Comment:
This needs to use the interner in the new rustbuild, see the ARMHF line above. Looks like it changed underneath you!